resources/js/Pages/Results.vue

```diff
@@ -2,6 +2,9 @@
     <div class="page-container results-page">
         <Head title="Mismatch Finder - Results" />
         <wikit-button class="back-button" @click.native="() => $inertia.visit('/')">
+            <template #prefix>
+                <icon type="arrowprevious" size="medium" color="inherit" dir="ltr"/>
```

Comment:
Here we can derive the language from the body, we just need to add `dir="auto"` to the vody tag and use `window.getComputedStyles(document.body).dir`. See Proof of concept at: https://codepen.io/xumium/pen/PoKQGQL

Reply:
Good one!! Tho I think it should be in the `<html>` tag, not in the body one. See [this link](https://www.w3.org/International/questions/qa-html-dir)